utils/common/src/main/kotlin/ProcessCapture.kt

```diff
@@ -59,10 +59,10 @@ class ProcessCapture(vararg command: String, workingDir: File? = null, environme
     }
 
     private val tempDir = createTempDirectory(javaClass.simpleName).toFile().apply { deleteOnExit() }
-    private val tempPrefix = command.first().substringAfterLast(File.separatorChar)
 
-    private val stdoutFile = tempDir.resolve("$tempPrefix.stdout").apply { deleteOnExit() }
-    private val stderrFile = tempDir.resolve("$tempPrefix.stderr").apply { deleteOnExit() }
+    private val commandName = command.first().substringAfterLast(File.separatorChar)
```

Comment:
Nit: Let's also drop empty line 62 while at it, as all 4 lines actually belong together.